src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup.mdx

```diff
@@ -154,87 +179,21 @@ Click on the link below for your language and complete the configuration steps.
    * [Swift](https://github.com/open-telemetry/opentelemetry-swift/tree/master/Sources/Exporters/OpenTelemetryProtocol)
    * [...Find additional OTLP language support in GitHub](https://github.com/open-telemetry)
 
-<CollapserGroup>
-  <Collapser
-    className="freq-link"
-    id="collector"
-    title="Export data to an OpenTelemetry Collector (optional) "
-  >
-The [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) is a configurable and extensible software component to receive, process, and export telemetry data. When you set up a collector, it can operate as a gateway or as an agent:
-
-* **Gateway:** The collector receives data from a variety of sources and applies standard processing before exporting to some backend.
-* **Agent:** The collector is deployed on each host in an environment and can collect telemetry data about the host and processes running on it.
-
-When you use a collector, you start by following the same routine as above for setting up OTLP in your service. In this case, instead of exporting data directly to New Relic, you export through a collector that you set up. In the collector, you configure the [OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter) to export data to New Relic.
-
-When your data goes through a collector, the transport looks like this:
-
-![Diagram showing OpenTelemetry using the OpenTelemetry Collector and New Relic's OTLP endpoint.](./images/native_otlp_with_collector.svg "Diagram of OTLP with collector")
-
-Here's a Docker example of how to set up and run an OpenTelemetry collector with the collector YAML:
-
-1. Save the following as `otel-config.yaml`:
-
-```
-receivers:
-  otlp:
-    protocols:
-      grpc:
-      http:
-
-processors:
-  batch:
-
-exporters:
-  otlp:
-    endpoint: ${OTEL_EXPORTER_OTLP_ENDPOINT}
-    headers:
-      api-key: ${NEW_RELIC_LICENSE_KEY}
-
-service:
-  pipelines:
-    traces:
-      receivers: [otlp]
-      processors: [batch]
-      exporters: [otlp]
-    metrics:
-      receivers: [otlp]
-      processors: [batch]
-      exporters: [otlp]
-    logs:
-      receivers: [otlp]
-      processors: [batch]
-      exporters: [otlp]
-```
-2. Run the OpenTelemetry collector after you make the following changes:
-   * Replace <var>OTLP_ENDPOINT_HERE</var> with the appropriate [endpoint](#review-settings.
-   * Replace <var>YOUR_KEY_HERE</var> with your account's [license key](https://one.newrelic.com/launcher/api-keys-ui.launcher).
-
-```
-export OTEL_EXPORTER_OTLP_ENDPOINT=<var>OTLP_ENDPOINT_HERE</var>
-export NEW_RELIC_LICENSE_KEY=<var>YOUR_KEY_HERE</var>
-
-docker run --rm \
-  -e OTEL_EXPORTER_OTLP_ENDPOINT \
-  -e NEW_RELIC_LICENSE_KEY \
-  -p 4317:4317 \
-  -v "${PWD}/otel-config.yaml":/otel-config.yaml \
-  --name otelcol \
-  otel/opentelemetry-collector \
-  --config otel-config.yaml
-```
-  </Collapser>
-</CollapserGroup>
+### Export from a collector [#collector-export]
```

Comment:
This subsection seems a little out of the way, considering you say above to go off and do the Otel stuff elsewhere and then get back and go to step 5, so this is kind of in between those steps and I'd imagine easy to miss, for what it's worth.

Reply:
@zuluecho9, great comment! It sounds like you're saying that users might not see that in Step 4, they need to choose one of two options. If that's the issue, how about a mini-toc like this:

```
Now that you are familiar with the New Relic-specific configurations for exporting, follow the instructions for the way you want to export data:

  * [Export directly to New Relic from your apps](#direct-export)
  * [Export to New Relic from a collector](#collector-export)

```

Reply:
yeah, whatever you think is best. I was just afraid they'd go off and return and miss that content, so whatever you think is best